xbmc/cores/RetroPlayer/RetroPlayerAutoSave.cpp

```diff
@@ -47,7 +48,7 @@ void CRetroPlayerAutoSave::Process()
     if (m_bStop)
       break;
 
-    if (m_player->GetSpeed() > 0.0f)
+    if (CDataCacheCore::GetInstance().GetSpeed() > 0.0f)
```

Comment:
@garbear btw: I won't use the generic IPlayer interface for this case where you know the type. you can implement a GetSpeed method for RP if you need one. GetSpeed in IPlayer is gone